database/influxdb/influxdb_user.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+
+# (c) 2016, Adam Hamsik <haaaad () gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: influxdb_user
+short_description: Manage InfluxDB users
+description:
+    - Manage InfluxDB users
+version_added: 2.3
+author: "Adam Hamsik (@haad)"
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
+options:
+    hostname:
+        description:
+            - The hostname or IP address on which InfluxDB server is listening.
+        required: true
+    username:
+        description:
+            - Username that will be used to authenticate against InfluxDB server.
+        default: root
+        required: false
+    password:
+        description:
+            - Password that will be used to authenticate against InfluxDB server.
+        default: root
+        required: false
+    port:
+        description:
+            - The port on which InfluxDB server is listening.
+        default: 8086
+        required: false
+    database_name:
```

Comment:
I'm not sure I understand the purpose of the `database_name` option. When specified, it doesn't seem to apply any grants to the user being created. For example, I created a non-admin user `test` with `database_name=ansible-test`, but they don't have any grants:

   ```
> show grants for test
database	privilege
--------	---------

>
```

If the goal is to be able to set specific grants for users, you may want to consider adding a `privilege` option (leveraging function [here](https://github.com/influxdata/influxdb-python/blob/master/influxdb/client.py#L708)).